.github/workflows/test_bwc.yml

```diff
@@ -0,0 +1,45 @@
+name: BWC
+on:
+  push:
+    branches:
+      - "**"
+  pull_request:
+    branches:
+      - "**"
+
+jobs:
+  Build-ff-linux:
+    strategy:
+      matrix:
+        java: [11,17,21]
+      fail-fast: false
+
+    name: Test Flow Framework BWC
+    runs-on: ubuntu-latest
```

Comment:
We should at least add windows platform too here.

Reply:
I tried, script doesn't work as is and would need tweaking:
```
Run plugin_version=`./gradlew properties -q | grep "opensearch_build:" | awk '{print $2}'`
plugin_version=./gradlew: D:\a\_temp\a25bdbdb-202a-4aa2-bc52-9ebd85a8cd4a.ps1:2
Line |
   2 |  plugin_version=`./gradlew properties -q | grep "opensearch_build:" |  …
     |  ~~~~~~~~~~~~~~~~~~~~~~~~~
     | The term 'plugin_version=./gradlew' is not recognized as a name of a cmdlet, function, script file, or
     | executable program. Check the spelling of the name, or if a path was included, verify that the path is correct
     | and try again.
```
This PR is not about BWCs, can we please move all requests to change BWC behavior into #555?

Reply:
It's fine to move it to #555. We should have had a separate PR for adding BWC tests in the project.